ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests.hs

```diff
@@ -1,9 +1,11 @@
 module Test.Consensus.Genesis.Tests (tests) where
 
 import qualified Test.Consensus.Genesis.Tests.LongRangeAttack as LongRangeAttack
+import qualified Test.Consensus.Genesis.Tests.Uniform as Uniform
 import           Test.Tasty
 
 tests :: TestTree
 tests = testGroup "Genesis tests"
     [ LongRangeAttack.tests
+    , Uniform.tests
```

Comment:
Minor: we use both leading and trailing commas. Shouldn't we stick to one style? (we prefer leading commas).

Reply:
No particular opinion on our side. Do you want us to change all the commas to leading wherever possible? Is there any tool that could take care of this (and/or other code styles you would want to enforce) automatically?